packages/manager/modules/vrack/src/vrack-associated-services/ipv6/ipv6.controller.js

```diff
@@ -21,7 +21,33 @@ export default class VrackAssignedIpCtrl {
   }
 
   $onInit() {
-    this.refreshData();
+    this.vrackAssignedIpv6Service
```

Comment:
should bridgedSubranges be instead in the resolve of the routing file?

Reply:
Nop because this component is not loaded at the init of the page doing like this allow to make the call only when we open the ipv6 detail